functional/sev/run.sh

```diff
@@ -0,0 +1,441 @@
+#!/bin/bash
+# Copyright 2022 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -eE
+set -o nounset
+set -o pipefail
+
+trap cleanup EXIT
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+tests_repo_dir="$(realpath "$script_dir/../..")"
+
+export WORKING_DIR="${HOME}/work"
+
+export EDK2_REPO_DIR="${HOME}/edk2"
+export DOCKER_USER="rksavino"
+
+export GOROOT="/usr/local/go"
+export GOPATH="${WORKING_DIR}/go"
+export PATH="${PATH}:${GOROOT}/bin"
+
+export KBS_DB_HOST="127.0.0.1"
+export KBS_DB_USER="root"
+export KBS_DB_PW="root"
+export KBS_DB="simple_kbs"
+export KBS_DB_TYPE="mysql"
+
+esudo() {
+  sudo -E PATH=$PATH "$@"
+}
+
+build_and_install_kata_runtime_4270() {
+  cd ${WORKING_DIR}
+
+  # Clone kata-containers repo and checkout PR 4270
+  git clone https://github.com/kata-containers/kata-containers.git
+  cd kata-containers
+  gh pr checkout 4270
```

Comment:
Is the intent to replace this with the actual PR being tested?

Reply:
This checkout of PR 4270 is temporary until it gets merged. It's close to being merged, but until then, wanted to get CI running.

Reply:
@ryansavino, `gh pr checkout` implies that we have https://github.com/cli/cli installed in machine, which may not be the case.

A simple `git checkout` would be a better way to avoid the dependency on the tool.

Reply:
https://github.com/kata-containers/kata-containers/pull/4270 is now merged. This part removed.